data/site.yml

```diff
@@ -14,5 +14,7 @@ contact_email: "info@euroteamoutreach.org"
 steele_email: "josh@euroteamoutreach.org"
 beal_email: "jbeal4ua@gmail.com"
 denise_email: "hutchisondenise@gmail.com"
+
+content_server_url: "https://d21yo20tm8bmc2.cloudfront.net"
```

Comment:
Let's not use a CDN for now. I did that on my blog because I had lots of content going back several years. For now, let's just store images locally under `source/assets/images/` In order to keep blog images separate from site images, lets use the same year-based folder structure that I have on my CDN. We can store everything under `/source/assets/images/blog/`, and in there have a new folder for each year.